hiive/models/staging/stg_customer_order_summary.sql

```diff
@@ -0,0 +1,67 @@
+{{
+    config(enabled=false)
+}}
+
+
+WITH raw_customers AS (
+   SELECT
+        id AS customer_id,
+        first_name,
+        last_name,
+        email_address,
+        phone_number,
+        created_at AS signup_date
+   FROM raw.database.orders
+   -- Select all customers regardless of status or signup date for simplicity
+),
+
+-- Get order data
+raw_orders AS (
+   SELECT
+       o.id AS order_id,
+       o.customer_id,
+       o.order_date,
+       o.order_status,
+       o.total_amount
+   FROM {{ source('raw', 'orders') }} o
```

Comment:
Since this model is potentially a data mart, avoid using sources. It's better to reference some staging model here.